src/nvim/spell.c

```diff
@@ -4292,23 +4290,22 @@ static void suggest_trie_walk(suginfo_T *su, langp_T *lp, char_u *fword, bool so
                     && utf_iscomposing(utf_ptr2char(fword
                                                     + sp->ts_fcharstart))) {
                   sp->ts_score -= SCORE_SUBST - SCORE_SUBCOMP;
-                } else if (!soundfold
-                           && slang->sl_has_map
+                } else if (!soundfold && slang->sl_has_map
```

Comment:
Do not join lines like this, lengthy multipart condition is more readable when each part starts on its own line.